hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/VolumeManagerImpl.java

```diff
@@ -255,6 +328,19 @@ public void setQuota(String volume, long quota) throws IOException {
     }
   }
 
+  @Override
+  public void applySetQuota(OmVolumeArgs omVolumeArgs) throws IOException {
+    try {
+      String dbVolumeKey = metadataManager.getVolumeKey(
```

Comment:
We should not need to do a read in applySetQuota. Can we pass the dbVolumeKey from start to apply?

Reply:
This is not DB reading, this method applys "/" before volume.